src/opnsense/service/conf/actions.d/actions_dns.conf

```
@@ -3,6 +3,7 @@ command:/usr/local/sbin/pluginctl -c dns_reload
 parameters:
 type:script_output
 message:rewriting /etc/resolv.conf and /etc/hosts
+description:Reload local DNS resolver configuration
```

```suggestion
description:Reload system DNS resolver configuration
```

related change for clarity: ed6cd638ac7e24